integration-libs/punchout/root/facade/punchout.facade.ts

```diff
@@ -39,7 +43,9 @@ export abstract class PunchoutFacade {
    * Abstract method used to get Punchout Session Requisition data
    * @param sessionId is the sesssion Id given by ARIBA via url param
    */
-  abstract getPunchoutSessionRequisition(
-    sessionId: string
```

Comment:
No need to provide the sessionId as parameter, the facade method now gets it from the state (within punchoutStoreService ) directly.